### PR TITLE
feat: add tickets notification

### PR DESCRIPTION
**Pull request by davideimola**

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -215,6 +215,11 @@ export default function Header() {
           {notificationOpen === true ? (
             <div className="notification-menu">
               <ul>
+                <Link href="https://www.eventbrite.it/e/open-source-day-2023-tickets-441134303577">
```
Comment (TheJoin95) [inline components/Header.tsx:218]: <Link target="_blank" href="https://www.eventbrite.it/e/open-source-day-2023-tickets-441134303577">

Comment (TheJoin95) [reply components/Header.tsx:218]: Il link a siti esterni deve aprirsi in un'altra pagina

**Comment by TheJoin95** [comment]
@davideimola don't forget to update the PR, thanks

**Review by TheJoin95 — APPROVED** [review]

**Comment by davideimola** [comment]
@TheJoin95 done it

**Comment by TheJoin95** [comment]
I see you, I approved ty

**Pull request merged**